relay-general/src/protocol/schema.rs

```diff
@@ -0,0 +1,5 @@
+use crate::protocol::Event;
+
+pub fn event_json_schema() -> impl serde::Serialize {
```

Comment:
_nit_: Since this is exposed from the crate, could you add a doc string for this?
Separatenly, I'm assuming that it's not possible to name this type, is it?

Reply:
we can name the type but then we'd have to reexport it from schemars. I guess the other way we can do this without exposing too much api surface is to write a newtype that impls serialize, but I don't think it's worth it?

Reply:
Probably not, you're right. As an idea: If all you intend to do is serialize the schema, though, you could consider implementing a `serialize_schema<T: Schema, S: Serializer>(serializer: S)` function or similar.

Exposing `impl Serialize` is definitely fine for now, this can still be changed in the future once we explore more requirements.